google-cloud-spanner/src/main/java/com/google/cloud/spanner/Struct.java

```diff
@@ -41,7 +43,9 @@
  * <p>{@code Struct} instances are immutable.
  */
 @Immutable
-public abstract class Struct extends AbstractStructReader {
+public abstract class Struct extends AbstractStructReader implements Serializable {
```

Comment:
`GrpcStruct` is serializable. Since Struct's constructor is package protected, this should be fine.